src/components/Requirements/IncompleteSubReqCourse.vue

```diff
@@ -81,6 +88,9 @@ export default Vue.extend({
     seeAll() {
       return 'See all >';
     },
+    coursesWithoutRequirementID() {
+      return this.courses.map(({ requirementID, ...rest }) => rest);
```

Comment:
try to suppress it using `return this.courses.map(({ requirementID: _, ...rest }) => rest);`?